src/systems/physics/Physics.cc

```diff
@@ -1409,6 +1409,15 @@ void PhysicsPrivate::UpdatePhysics(EntityComponentManager &_ecm)
             this->entityWorldVelocityCommandMap);
         if (!worldAngularVelFeature)
         {
+          static bool informed{false};
```

Comment:
What is the purpose of `informed` here? Just so the debug message is only printed once?

Reply:
Yup, this is the pattern we've been using throughout this file to avoid spamming the user.